### DISCUSSION TITLE
Problem with auth in Flutter

### DISCUSSION THREAD

**Discussion by IgnacioNicolasCano** [supabase/supabase#19614] [Questions]
I was working on an app with accounts, but one day it stops working in Android. The problem is related with auth. The error is: "No host specified in URI /auth/v1/token?grant_type=password". I have try changing the Dart and Flutter SDK and flutter_supabase version. If you response try to explain it very detailed because I'm new on this. 

I normally use VSCode, but this screenshot is from Android Studio:
![Captura de pantalla 2023-12-12 021041](https://github.com/supabase/supabase/assets/129014779/f7a78d03-e8db-4c6c-a75f-d06388ddff96)

Thanks.

**Comment by GaryAustin1** [comment]
I don't know flutter but your error seems to be saying you are missing the Supabase instance URL (like https://ugcxxxxxxxxxxxxxbq.supabase.co/).  This implies you did not provide a URL in createclient... .but this is just a guess....

**Reply by IgnacioNicolasCano** [reply]
I have provided, everything works, but auth

**Reply by IgnacioNicolasCano** [reply]
I have solved the problem, I have it in a .env and I didn't include it correctly in pubspec.yaml
